lib/app/features/auth/views/pages/recover_user_page/components/recovery_creds_step.dart

```diff
@@ -15,9 +15,15 @@ class RecoveryCredsStep extends ConsumerWidget {
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
-    final recoverUserState = ref.watch(recoverUserActionNotifierProvider);
+    final isLoading = ref.watch(
+          initUserRecoveryActionNotifierProvider.select((it) => it.isLoading),
+        ) ||
+        ref.watch(
+          completeUserRecoveryActionNotifierProvider.select((it) => it.isLoading),
+        );
+
```

Comment:
Move providers return values to variables to have more readability in the condition.

Reply:
Done.